src/js/BookReader/events.js

```diff
@@ -17,4 +17,5 @@ export const EVENTS = {
   /* currently 3 represents thumbnail view */
   '3PageViewSelected': '3PageViewSelected',
   mobileNavOpen: 'mobileNavOpen',
+  pageChanged = 'pageChanged'
```

Comment:
hi, i believe this will break the object.

Reply:
Ok, making the required changes. Thanks

Reply:
@iisa I've made the required change, please review it.
Thanks.